class ValidationError(Exception):
    pass

class ConflictError(Exception):
    pass

class ResourceNotFound(Exception):
    pass
